lib/ansible/module_utils/facts.py

```diff
@@ -99,6 +99,7 @@ class Facts(object):
                     ('/etc/os-release', 'SuSE'),
                     ('/etc/gentoo-release', 'Gentoo'),
                     ('/etc/os-release', 'Debian'),
+                    ('/etc/debian_version', 'Debian'),
```

Comment:
Not, it's not the same format. Actually it's very unhelpful:

 ```
root@aca8dab98b30:/# cat /etc/debian_version 
6.0.8
root@aca8dab98b30:/# cat /etc/os-release
cat: /etc/os-release: No such file or directory
```

While on Wheezy:

```
root@1db7bbc340b3:/# cat /etc/debian_version 
7.6
root@1db7bbc340b3:/# cat /etc/os-release     
PRETTY_NAME="Debian GNU/Linux 7 (wheezy)"
NAME="Debian GNU/Linux"
VERSION_ID="7"
VERSION="7 (wheezy)"
ID=debian
ANSI_COLOR="1;31"
HOME_URL="http://www.debian.org/"
SUPPORT_URL="http://www.debian.org/support/"
BUG_REPORT_URL="http://bugs.debian.org/"
root@1db7bbc340b3:/# 
```

Basically I'm using /etc/debian_version only so I have a file which tells me it's a Debian box and later down I can run the lsb commands.


Thoughts?